service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/endpoints/RestaurantRoutes.kt

```diff
@@ -0,0 +1,30 @@
+package org.thechance.service_restaurant.api.endpoints
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import org.koin.ktor.ext.inject
+import org.thechance.service_restaurant.api.models.mappers.toDto
+import org.thechance.service_restaurant.api.usecases.RestaurantCasesContainer
+
+fun Route.restaurantRoutes() {
+
+    val restaurant: RestaurantCasesContainer by inject()
+
+    route("/restaurant") {
```

Comment:
Change restaurant to restaurants

Reply:
@devsadeq @aya-seif This already changed in my branch to "restaurantUcesCases" and merged on develop.